.github/actions/lint/action.yml

```diff
@@ -35,10 +38,32 @@ runs:
     shell: bash
     run: |
       set -e -o pipefail
-
-      # On pull requests, HEAD^1 will always be the merge base, so consider that diff for formatting.
```

Comment:
Same here, the comments are not related to the Flutter linting, so they can remain intact :)